### ISSUE TITLE
WS2812B + Arduino Uno + Adalight + Prismatik Screen Capture - flashing issue

### ISSUE THREAD

**Issue by PetrLukes** [FastLED/FastLED#1135]
Hello,

I have setup mentioned above, and after around 10 mins of usage, I have constant flashing same as on video below
https://www.youtube.com/watch?v=xHdZHHp_NBI&ab_channel=PetrLuke%C5%A1

I think it's not Prismatik related, since the flashing is happening also when the program is not running.
I'm using the latest version of FastLED with Adalight modification.

Any sort of help is grealty appreciated

**Comment by Benik3** [comment]
I recommend to try to use NeoPixelBus library with Uart ("non blocking") method :)
https://github.com/Benik3/Adalight_WS2812_ESP8266_Non-blocking

**Comment by PetrLukes** [comment]
Unfortunately, I cant make this version of NeoPixelBus work. Now I'm using FASTLED

**Comment by sblantipodi** [comment]
@PetrLukes I would try the Luciferin suite.
It consists of a firmware and a PC software, it does not have those problems.

https://github.com/sblantipodi/firefly_luciferin

**Comment by Benik3** [comment]
> 
> 
> Unfortunately, I cant make this version of NeoPixelBus work. Now I'm using FASTLED

Yeah I got that you are using Fastled, but I wanted to see the whole code...
I use Arduino micro with FastLed and APA102 + Prismatik for more then 3 years without problem.
Also what you didn't get to work on the NeoPixelBus? 

@sblantipodi I don't see firmware for UNO, does it support only ESP?

**Comment by zackees** [comment]
Reopening: this issue was closed automatically by a stale-bot run, not by a maintainer decision. That auto-close behavior was a mistake and has been disabled — FastLED issues will no longer be closed for inactivity. Apologies for the noise.
